tensorflow/c/kernels_test.cc

```diff
@@ -331,6 +332,7 @@ TEST(TestKernel, TestHostMemory) {
   device_type: "FakeDeviceName1"
   host_memory_arg: "input2"
   host_memory_arg: "output1"
+  subdevice_type: "FakeDeviceName1"
```

Comment:
super nit. May be use a different name for subdevice, for e.g. "FakeSubDeviceName1".  Just in case device_type gets treated as subdevice_type due to an error in the future.

Reply:
This is expected behavior to have same name for device and sub-device in this case, because this case will directly call TF_NewKernelBuilder without device registered, subdevice type will be set as the same string as device type.

Reply:
Makes sense.
It would be nice to have a test when the types are different, but may be it is easier to do when the rest of pluggable device implementation lands.

Reply:
ok, we will add it together with pluggable device implementation PR